java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -2527,7 +2527,7 @@ public final ColumnVector getMapValue(Scalar key) {
   }
 
   /** For a column of type List<Struct<String, String>> and a passed in String key, return a boolean
-   * scalar for all keys in the structs, true if the key exists in all maps in the column.
+   * column for all keys in the structs, true if the key exists in all maps in the column.
```

Comment:
This is still incorrect.  It says "true if the key exists in **all** maps in the column" which is incorrect.  It is true if the key exists in the corresponding map for that row, false otherwise.  It will never return null for a row.

Reply:
Done.